.gitignore

```diff
@@ -272,16 +272,26 @@ miniapps/navier/*_output
 
 miniapps/nurbs/nurbs_ex1
 miniapps/nurbs/nurbs_ex1p
+miniapps/nurbs/nurbs_ex3
+miniapps/nurbs/nurbs_ex5
 miniapps/nurbs/nurbs_ex11p
+miniapps/nurbs/nurbs_ex24
+miniapps/nurbs/nurbs_solenoidal
 miniapps/nurbs/nurbs_printfunc
 miniapps/nurbs/nurbs_patch_ex1
 miniapps/nurbs/nurbs_curveint
 miniapps/nurbs/refined.mesh
 miniapps/nurbs/mesh.*
+miniapps/nurbs/sol_?.gf
```

Comment:
Does `sol_?.gf` actually exist?

Reply:
These outputs are generated by nurbs_ex5 and nurbs_solenoidal.

As such I assumed it would be create during `make test`. This is not  entirely correct, because they also get deleted. So in that sense the line could be deleted. But so could the other lines in the ignore file???

I do not know what the mfem convention is? But my suggested for now would be to add this output to the ignore file.